Source/WebDriver/socket/SessionHostSocket.cpp

```diff
@@ -30,10 +30,21 @@
 #include <wtf/UUID.h>
 #include <wtf/text/MakeString.h>
 
+#if PLATFORM(WIN)
+#include <shlwapi.h>
+#include <ws2tcpip.h>
+#endif
+
 namespace WebDriver {
 
 SessionHost::~SessionHost()
 {
+#if PLATFORM(WIN)
+    if (m_browserHandle) {
+        ::TerminateProcess(*m_browserHandle, 0);
```

Comment:
You need `CloseHandle` here. It's better to use `Win32Handle` for `m_browserHandle`.

Reply:
I see. I'll try it.